### ISSUE TITLE
lychee-action produces no output and doesn't seem to do anything

### ISSUE THREAD

**Issue by alebcay** [lycheeverse/lychee-action#24]
I've tried using lychee-action v1.0.7 or v1.0.8 and see runs like [this](https://github.com/alebcay/awesome-shell/runs/2412855766?check_suite_focus=true) happening. It looks like nothing is actually running as part of the action?

The workflow file is here:
```
name: GitHub Actions CI
on:
  push:
    branches: master
  pull_request:

jobs:
  check_links:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Check links
        uses: lycheeverse/lychee-action@v1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

      - run: cat ./lychee/out.md
        if: ${{ always() }}
```

Not sure if I'm just doing something wrong or if there has been some kind of a regression.

**Comment by mre** [comment]
Yeah it's a regression. It is caused by #23, as input files have to be at the end. :( Will revert that change and then work on a fix. Thanks for the heads up.
@ltalirz fyi.

**Comment by mre** [comment]
Ah no, I think I was wrong. You use `lychee-action@v1`, which doesn't support proper output handling yet.
Can you try with `lycheeverse/lychee-action@v1.0.8` and report back?
If you found `lychee-action@v1` being mentioned any where, we'll have to update the docs. 🤷

**Comment by mre** [comment]
For clarification, I can only see v1 in your CI, e.g. here:
https://github.com/alebcay/awesome-shell/actions/runs/775129251/workflow
Can you try to explicitly use `lycheeverse/lychee-action@v1.0.8` or `lycheeverse/lychee-action@master` to see if that fixes the problem?

**Comment by ltalirz** [comment]
Just to mention that it's of course possible that something broke in the PR #23 - but I can confirm at least that v1.0.8 solved my issue and ran fine for my use case.

**Comment by alebcay** [comment]
I see - let me try again with `lycheeverse/lychee-action@v1.0.8`. I did not find `lychee-action@v1` anywhere, I just tried that and assumed it would choose the latest in 1.x.x (that may be a misunderstanding of how GitHub Actions versioning works on my part).

**Comment by alebcay** [comment]
Just tried and confirmed specifying v1.0.8 fixed it. Interestingly enough I get no output when I run the action (even v1.0.8 specifically) in https://github.com/nektos/act locally, but I suspect that may be a bug with `act` rather than this action (seeing as it works fine in GitHub itself). Thanks for the prompt responses!
